docs/testing.rst

```diff
@@ -8,148 +8,176 @@
 Front-end testing
 =================
 
-Bedrock runs a suite of front-end functional tests using `CasperJS`_, which is powered by
-a headless web browser called `PhantomJS`_. We also have an additional suite of `Jasmine`_
-behavioral/unit tests, which use `Karma`_ as a test runner. All these test suites live in
-the ``tests`` directory.
+Bedrock runs a suite of front-end `Jasmine`_ behavioral/unit tests, which use
+`Karma`_ as a test runner. We also have a suite of functional tests using
+`Selenium`_ and `pytest`_. This allows us to emulate users interacting with a
+real browser. All these test suites live in the ``tests`` directory.
 
 The ``tests`` directory comprises of:
 
-* ``/functional`` contains individual CasperJS test suites.
+* ``/functional`` contains pytest tests.
+* ``/pages`` contains Python page objects.
 * ``/unit`` contains the Jasmine tests and Karma config file.
-* ``/lib`` contains various ``config`` and ``helpers`` functions used to run the functional tests.
-* ``config.json`` a configuration file used by PhantomJS when running tests.
 
 Installation
 ------------
 
-The specific versions of CasperJS, PhantomJS and Jasmine/Karma which are needed to run the
-tests, can be installed by running the following command from the root directory of the project::
+First follow the :ref:`installation instructions for bedrock<install>`, which
+will install the specific versions of Jasmine/Karma which are needed to run the
+unit tests, and guide you through installing pip and setting up a virtual
```

Comment:
peep doesn't need to be installed as it's in bedrock/bin, and peep depends on pip, so that does still need to be installed.